cognite/client/_api/data_modeling/instances.py

```diff
@@ -558,19 +591,34 @@ def _load(
                     node_cls._load(data) if data["instanceType"] == "node" else edge_cls._load(data)
                     for data in resource_list
                 ]
-                return cls(resources, None)
+                return cls(resources, None, None)
+
+            @classmethod
+            def _load_raw_api_response(
+                cls, responses: list[dict[str, Any]], cognite_client: CogniteClient
+            ) -> _NodeOrEdgeList:
+                typing: TypeInformation | None = None
+                if len(responses) >= 1:
+                    typing = TypeInformation._load(responses[0]["typing"]) if "typing" in responses[0] else None
```

Comment:
Same suggestion here as above. Any way we don't have to re-implement this method?